client-libraries/c/Cargo.toml

```
@@ -0,0 +1,41 @@
+[package]
+name = "modality-capi"
+version = "0.1.0"
+edition = "2021"
+authors = ["Jon Lamb <jon@auxon.io>"]
+license = "Apache-2.0"
+repository = "https://github.com/auxoncorp/modality-sdk"
+description = "Part of Auxon's Modality SDK"
```

Heh, I just removed this description from all the rust libs. Can we:

```suggestion
description = "The C & C++ FFI for Auxon's Modality SDK"
```

---

Sounds good, will do